### PR TITLE
Move manage contacts setting page up a level

### PR DESCRIPTION
**Pull request by buoyad**
Out of settings > account into the main list here:
image

cc @keybase/y2ksquad

### CODE REVIEW

File: shared/settings/index.tsx [hunk 1]
```diff
@@ -44,6 +45,7 @@ const mergeProps = (stateProps: ReturnType<typeof mapStateToProps>, dispatchProp
   badgeNotifications: stateProps.badgeNotifications,
   badgeNumbers: stateProps._badgeNumbers.toObject(),
   children: ownProps.children,
+  contactsLabel: stateProps._contactImportEnabled ? 'Contacts' : 'Import contacts',
```
Comment (jakob223) [inline shared/settings/index.tsx]: Can you capitalize the C in `Import contacts` to match the other things in this menu?

File: shared/settings/nav/index.native.tsx [hunk 2]
```diff
@@ -54,7 +54,7 @@ function SettingsNav(props: Props) {
             {
               badgeNumber: props.badgeNumbers[TabConstants.settingsTab],
               onClick: () => props.onTabChange(Constants.accountTab),
-              text: 'Account',
+              text: 'Your account',
```
Comment (jakob223) [inline shared/settings/nav/index.native.tsx]: Can you capitalize `Your Account` to be consistent with the other stuff in this menu?

File: shared/settings/nav/index.stories.tsx [hunk 3]
```diff
@@ -4,6 +4,7 @@ import {action, storiesOf} from '../../stories/storybook'
 
 const defaultProps = {
   badgeNumbers: {},
+  contactsLabel: 'Import contacts',
```
Comment (jakob223) [inline shared/settings/nav/index.stories.tsx]: Cap C

Comment (buoyad) [reply shared/settings/nav/index.stories.tsx]: First letter only capitalized is actually more consistent with how we title most of the menu items / modal titles / buttons in the app. The "Your account" change was at Cécile's request (see that title on desktop in master)

**Comment by buoyad** [comment]
@jakob223 I made a change to the title copy and ordering after discussing with @cecileboucheron

File: shared/settings/nav/index.stories.tsx (hunk 3 above)
Comment (jakob223) [reply shared/settings/nav/index.stories.tsx]: 👌 thanks

File: shared/settings/nav/index.native.tsx (hunk 2 above)
Comment (jakob223) [reply shared/settings/nav/index.native.tsx]: Should `Screen Protector` below be `Screen protector` then? That's what made me think they should be capitalized.

**Pull request merged**